chat-android/src/main/java/com/ably/chat/RoomStatus.kt

```diff
@@ -118,3 +162,60 @@ data class RoomStatusChange(
      */
     val error: ErrorInfo? = null,
 )
+
+open class RoomStatusEvenEmitter : EventEmitter<RoomLifecycle, RoomStatus.Listener>() {
```

Comment:
here is typo in the name, should be `...EventEmitter`

Reply:
Fixed https://github.com/ably-labs/ably-chat-kotlin/pull/27/commits/ce9c0baf4085c8199771f89c29b3d60fe399db26